content/300-guides/100-performance-and-optimization/090-metrics.mdx

```diff
@@ -0,0 +1,335 @@
+---
+title: 'Metrics'
+metaTitle: 'Metrics'
+metaDescription: 'Diagnose application performance with insights into Prisma Client database activity'
+tocDepth: 3
+---
+
+<TopBlock>
+
+Prisma metrics give you a detailed insight into how Prisma Client interacts with your database. You can use this insight to help diagnose performance issues with your application. If you send the data to an external metrics system, then you can view the data over time. For example, you might visualize how your application's number of idle and active connections change.
+
+</TopBlock>
+
+## About metrics
+
+Metrics provide information on the following aspects of Prisma Client:
+
+- The number of Prisma operations
+
+  A Prisma operation is a statement like `let user = prisma.user(include: {posts: true}})`, which Prisma Client translates into one or more SQL queries.
+
+- Query response times
+- Active transactions
+- Connection pool information
+
+You can export metrics in JSON or Prometheus formats and view them in a console log, or integrate them into an external metrics system, such as StatsD or Prometheus. If you integrate them into an external metrics system, then you can view the metrics data over time, for example to help diagnose changes in your application's performance.
```

Comment:
> for example to help diagnose changes in your application's performance.

This feels a bit repetitive. Anyone familiar with Metrics should know this and we explained it above

Reply:
Fair point - to reduce the repetition, I've moved some of the top-of-page information into this section, and also reworked this material.